### ISSUE TITLE
Implement Annotation in GameBio (Post & Fetch)

### ISSUE THREAD

**Issue by elifkizilky** [bounswe/bounswe2023group7#782]
### Issue Description

I will implement post (create annotation) and get annotation for game bio in the game page as mentioned in #755 .

### Tasks

Parts of the task:
- [x] create annotation in game bio
- [x] get annotation in game bio

### Deadline of the Issue

25.12.23 15.00

### Reviewer

@mtkamiloglu 

### Deadline for the Review

25.12.23 18.00

**Comment by elifkizilky** [comment]
I added annotation to the game page for the game bio section. The rest of the sections will be implemented by my team mates as discussed. So I will leave some hints for them to implement:

- Create recogito annotator with useRef.
- For the game page specifc, check if the game exists since it will be fetched from the db first, then inside useEffect fetchAnnotations, and add event handler for creating annotations. Write the related functions, the codebase will be very similar to what I wrote.
- Destroy the annotator after returning.
- Implement display annotations function with the annotator you used in useRef. Use addAnnotation function in the recogito library.
